FlowCal/excel_ui.py

```diff
@@ -1324,11 +1324,11 @@ def run(input_path=None, output_path=None, verbose=True, plot=True):
      4. Generate statistics for each bead sample.
      5. Process all the cell samples in the Samples table.
      6. Generate statistics for each sample.
-     7. Generate a histogram table for each fluorescent channel specified
-        for each sample.
+     7. If specified, generate a histogram table for each fluorescent channel
+        specified for each sample.
      8. Generate a table with run time, date, FlowCal version, among
         others.
-     9. Save statistics and histograms in an output Excel file.
+     9. Save statistics and (if specified) histograms in an output Excel file.
```

Comment:
"requested" might be a better word than "specified" in this description, but this is minor